pkg/controllers/binding/binding_controller.go

```diff
@@ -194,7 +194,7 @@ func (c *ResourceBindingController) newOverridePolicyFunc() handler.MapFunc {
 			workload, err := helper.FetchResourceTemplate(c.DynamicClient, c.InformerManager, c.RESTMapper, binding.Spec.Resource)
 			if err != nil {
 				klog.Errorf("Failed to fetch workload for resourceBinding(%s/%s). Error: %v.", binding.Namespace, binding.Name, err)
-				return nil
+				continue
```

Comment:
Can here add some comment to explain why we can continue?

Reply:
Does `cluster_binding_controller.go` have the same problem?